core/src/main/kotlin/ch/ergon/dope/validtype/ValidType.kt

```diff
@@ -4,11 +4,13 @@ sealed interface ValidType
 
 interface ComparableType : ValidType
 
-interface BooleanType : ValidType
+interface AtomType : ValidType
```

Comment:
please update the diagram

Reply:
Why did you change that?

Reply:
@jansigi I changed it specifically for the `ToArrayExpression`, which behaves differently for atomic types (number, string, boolean). After a quick discussion with pgruntz I added the new interface 🙂

Reply:
> please update the diagram

done